provider/pkg/schema/gen.go

```diff
@@ -1442,3 +1446,5 @@ func GatherSemantics(schemaDir string) (SemanticsSpecDocument, error) {
 	}
 	return semanticsDocument, nil
 }
+
+var ExtensionResourceToken string = "aws-native:index:ExtensionResource"
```

Comment:
I don't see where this moved from?

Reply:
Ah, it's in the metadata.go .. and it's still there at this point, then it should be removed in the next PR. I also couldn't find where it had gone so fixed the compile error with this line.